lib/mix/lib/mix/tasks/release.init.ex

```diff
@@ -301,9 +301,10 @@ defmodule Mix.Tasks.Release.Init do
     setlocal enabledelayedexpansion
 
     pushd .
-    cd "%~dp0\.."
+    pushd "%~dp0\.."
     set RELEASE_ROOT=%cd%
     popd
+    popd
```

Comment:
Do we need this popd too or can we remove it?

```suggestion
```

Reply:
I don't know.
But seemed obvious for me that one pushd - one popd

Reply:
Tried a little script : 
```batchfile

cd 
pushd "c:\temp"
cd
pushd "d:\src"
cd
popd
cd
popd 
``` 

Without the second popd, I'm still in then C:\temp and not restored in the original directory where a launch it.

So I'l say Yes , we need the second too.